Pipfile

```diff
@@ -12,6 +12,7 @@ flake8 = "==3.8.3"
 pre-commit = "==2.15.0"
 twine = "*"
 bumpversion = "*"
+wheel = "*"
```

Comment:
:+1: 

Side note: It seems that since commit 5628321d we are maintaining both this `Pipfile` and a `requirements.txt`. For example `wheel` was in requirements and not in Pipfile. That's too bad! I wasn't involved in the CI but it would be great to use Pipenv everywhere and get rid of the requirements file. Here is a Github action for pipenv: https://github.com/marketplace/actions/pipenv-for-github-actions